packages/core/src/provider/error/rpcError.ts

```diff
@@ -0,0 +1,14 @@
+class RPCError extends Error {
```

Comment:
Can we use the RPC error types from `starknet-js`?

Reply:
yes we can - will add